menu.py

```diff
@@ -571,23 +571,30 @@ def menus():
     company_name = settings['-company-']
     if not company_name:
         company_name = "... <Bitte erst Server-Einstellungen setzen>"
+    last_window_location = tuple(sg.UserSettings().get('-last-window-location-', (None, None)))
     window = sg.Window(utils.title(),
                        layout,
                        default_element_size=(12, 1),
                        default_button_element_size=(12, 1),
+                       location=last_window_location,
                        finalize=True)
 
     # ------ Loop & Process button menu choices ------ #
     window.bring_to_front()
+    last_window_location = window.current_location(more_accurate=True)
```

Comment:
here I get an error: current_location() got an unexpected keyword argument `more_accurate`. Which version of PySimpleGUI do you use?

Reply:
The latest version I guess. I have set up erpnext-client from scratch on my laptop yesterday. `pip freeze` says
```
PySimpleGUI==4.47.0
```

Reply:
aha, indeed, 4.47.0 works. What is the Pythonic way to ensure that users upgrade to this version?

Reply:
I checked the release notes. This parameter was added in 4.47.0 and this version was released two days ago. I didn't realize this was brand-new.

About ensuring an upgrade, I think the version requirement should be added to the `requirements.txt`, and after every `git pull --rebase` `pip3 install -r requirements.txt` should be run. I'll add this.